src/components/Carousel.js

```diff
@@ -338,9 +341,10 @@ export default class Carousel extends Component {
       e.preventDefault();
       e.stopPropagation();
     }
+    const { changedTouches } = e;
     if (this.state.dragStart !== null) {
       this.setState(previousState => ({
-        dragOffset: e.changedTouches[0].pageX - previousState.dragStart,
```

Comment:
I don't think it will fix anything because `e.changedTouches` is the same for each callback call

Reply:
I think I don't get it.

I've tested in development using the project in which I had the problem, everything is working right since the changes were added. The problem is that inside this.setState, e was nullified.

Reply:
you mean not exactly `e` but `e.changedTouches` becomes null?
it would make sense

Reply:
`e` inside the setState function becomes null, so changedTouches is inaccessible, that is causing the crash